### PR TITLE
Reward quest 001

### PR DESCRIPTION
**Pull request by BahubaliPR**
Please review and give comments. If it is fine then merge it to master branch.

### CODE REVIEW

File: rewardquest/src/main/java/org/rewardquest/com/pages/LoginPage.java [hunk 1]
```diff
@@ -27,36 +23,61 @@
...
-		}
-		catch(Exception e) {
+			wait = new WebDriverWaits();
+		} catch (Exception e) {
 			e.printStackTrace();
 		}
 	}
 
 	/*
-	 * Clicks on DEMO SITES Clicks on Automation practice table Select column
```
Comment (gskr442) [inline rewardquest/src/main/java/org/rewardquest/com/pages/LoginPage.java:44]: Demo Sites and Automation Practice table ??

File: rewardquest/src/main/java/org/rewardquest/com/pages/LoginPage.java [hunk 2]
```diff
@@ -1,22 +1,18 @@
...
+import org.rewardquest.com.wait.WebDriverWaits;
+import org.testng.Assert;
 
 public class LoginPage {
 
-	public WebDriver driver = null;
-
+	private WebDriver driver = null;
+	private WebDriverWaits wait = null;
+	String textError;
```
Comment (gskr442) [inline rewardquest/src/main/java/org/rewardquest/com/pages/LoginPage.java]: Why no access modifier for "textError" variable ?

File: rewardquest/src/main/java/org/rewardquest/com/pages/LoginPage.java (hunk 1 above)
Comment (BahubaliPR) [reply rewardquest/src/main/java/org/rewardquest/com/pages/LoginPage.java:44]: Removed Unnecessary things like Demo Sites and Automation Practice table .

File: rewardquest/src/main/java/org/rewardquest/com/pages/LoginPage.java (hunk 2 above)
Comment (BahubaliPR) [reply rewardquest/src/main/java/org/rewardquest/com/pages/LoginPage.java]: Added access modifier for errorText variable.

**Review by gskr442 — APPROVED** [review]
Changes are fine.

**Pull request merged**